.github/workflows/publish-pypi.yml

```diff
@@ -4,10 +4,11 @@
 name: Upload Release Package to PyPI
 
 on:
-  release:
-    types: [published]
   workflow_dispatch:
-    inputs: {}
+    inputs:
+      release_version:
+        description: 'The semantic version number of the release (v[0-9].[0-9]+.[0-9]+)'
+        required: true
```

Comment:
does `true` work if it's lower case?

Reply:
Not exactly sure I understand, but it is using a regex to check the version number so it must be a lower case v. The provided version number is always required (until I find a better way for this action to scrape it).